superset-frontend/src/dashboard/components/DashboardBuilder/DashboardContainer.tsx

```diff
@@ -68,35 +67,29 @@ const DashboardContainer: FC<DashboardContainerProps> = ({
             the entire dashboard upon adding/removing top-level tabs, which would otherwise
             happen because of React's diffing algorithm
           */
-          <TabContainer
+          <Tabs
             id={DASHBOARD_GRID_ID}
-            activeKey={Math.min(tabIndex, childIds.length - 1)}
-            onSelect={handleChangeTab}
-            // @ts-ignore
-            animation
-            mountOnEnter
-            unmountOnExit={false}
+            activeKey={activeKey}
+            renderTabBar={() => <></>}
```

Comment:
Could just return null?

Reply:
`null` isn't allowed as the returned type, only `undefined`, but the behavior is different.

Reply:
Is this even needed if it's not rendering anything?

Reply:
Yes because `Tabs` renders a default `tabBar` with predefined height and margin as you can see below:

<img width="1126" alt="Screen Shot 2021-04-27 at 8 30 58 AM" src="https://user-images.githubusercontent.com/70410625/116234439-fbd25100-a732-11eb-8f64-dbee6a0202e0.png">